src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2550 @@
+    response = requests.get("http://localhost:8000/api/v1/battle-assistant/settings/gamepad-types")
+    types = response.json()["gamepad_types"]
+    for gamepad_type in types:
+        print(f"{gamepad_type['display_name']}: {gamepad_type['description']}")
+    ```
+    """
+    try:
+        gamepad_types = [
+            GamepadTypeInfo(
+                value="none",
+                display_name="无手柄",
+                description="不使用手柄，仅使用键盘操作",
```

Ruff (0.12.2)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)